addons/hr_recruitment/views/res_config_settings_views.xml

```diff
@@ -1,6 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
+        <record id="open_job_board_modules" model="ir.actions.act_window">
```

Comment:
Follow the naming guidelines. You marked the comment as resolved but it's not applied.

See: https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#xml-ids-and-naming

Note: Could be applied on the whole PR.